_data/payments.yml

```diff
@@ -27,8 +27,10 @@ websites:
       url: https://www.dwolla.com
       twitter: dwolla
       img: dwolla.png
-      tfa: No
-      status: https://twitter.com/dwolla/status/611900566281822208
+      tfa: Yes
+      software: Yes
+      status: https://twitter.com/dwolla/status/631497394538987520
```

Comment:
You don't need `status:` if it has 2fa support. It's designed only for "working on it" situations. Please remove this line.